### PR TITLE
Fix public image build

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Public Image cloud-governance pypi package is 1.1.306, however it should be 1.1.307.

### CODE REVIEW

File: .github/workflows/Build.yml [hunk 1]
```diff
@@ -313,8 +313,6 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: ⌛ Upload to 🐋 quay.io
         run: |
-          pip install setuptools
-          version=$(python3 setup.py --version)
```
Comment (ebattat) [inline .github/workflows/Build.yml:317]: Why did u delete the version ?

**Review by ebattat — COMMENTED** [review]
Pls remove the extra enter:
https://github.com/redhat-performance/cloud-governance/blob/67e0424aa46790be77657bedc508c0a941810b66/.github/workflows/Build.yml#L298-L299

File: .github/workflows/Build.yml (hunk 1 above)
Comment (athiruma) [reply .github/workflows/Build.yml:317]: we always use the latest version, so there is no need for it.

**Comment by ebattat** [comment]
So let remove also:
[link](https://github.com/redhat-performance/cloud-governance/blob/67e0424aa46790be77657bedc508c0a941810b66/.github/workflows/Build.yml#L342)
https://github.com/redhat-performance/cloud-governance/blob/67e0424aa46790be77657bedc508c0a941810b66/.github/workflows/Build.yml#L339

**Comment by athiruma** [comment]
> So let remove also:
> 
> https://github.com/redhat-performance/cloud-governance/blob/67e0424aa46790be77657bedc508c0a941810b66/.github/workflows/Build.yml#L342
> 
> 
> https://github.com/redhat-performance/cloud-governance/blob/67e0424aa46790be77657bedc508c0a941810b66/.github/workflows/Build.yml#L339

here we should do, as because for the public image no need to pass the version, just latest is enough.

**Review by ebattat — COMMENTED** [review]
The reason for specifying the Quay version is to allow reverting in case of a major error

**Comment by athiruma** [comment]
> The reason for specifying the Quay version is to allow reverting in case of a major error

right, but this is a public quay. we want to maintain only latest version.

File: .github/workflows/Build.yml (hunk 1 above)
Comment (athiruma) [reply .github/workflows/Build.yml:317]: We are publishing to the public quay so build and push must be there.
I don't find extra spaces.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**